hack/jenkins/e2e.sh

```diff
@@ -650,6 +650,7 @@ case ${JOB_NAME} in
     : ${E2E_TEST:="false"}
     : ${USE_KUBEMARK:="true"}
     : ${NUM_MINIONS:="40"}
+    : ${E2E_ZONE:="asia-east1-a"}
```

Comment:
This isn't going to work. `E2E_ZONE` is already defined on L71, so this won't override.